src/Esfa.Vacancy.UnitTests/CreateApprenticeship/Application/GivenACreateApprenticeshipParametersMapper/WhenMappingTrainingDetails.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Esfa.Vacancy.Application.Commands.CreateApprenticeship;
+using Esfa.Vacancy.Domain.Entities;
+using FluentAssertions;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using Ploeh.AutoFixture;
+using Ploeh.AutoFixture.AutoMoq;
+
+namespace Esfa.Vacancy.UnitTests.CreateApprenticeship.Application.GivenACreateApprenticeshipParametersMapper
+{
+    [TestFixture]
+    public class WhenMappingTrainingDetails
+    {
+        [Test]
```

Comment:
What is the expectation for these tests?  Do we need to add a Then.... to the method name?

Reply:
fixed